### DISCUSSION TITLE
Age data for EBS and NBS for Pcod, yellowfin, and pollock ages added to RACEBASE since February 2022

### DISCUSSION THREAD

**Discussion by NancyRoberson** [afsc-gap-products/model-based-indices#46] [General]
Hi ModSquad,

Lewis asked if I could post new age data sets as they are added to RACEBASE for EBS and NBS pollock, Pcod, and yellowfin sole. 
Please find attached the data sets that have been added since February 2022.  Note:  the 2022 EBS Pcod data set has only one vessel aged so far and the other vessel's ages should arrive sometime soon.
Thanks and I will be more diligent in reporting - it completely fell off my radar so I apologize for that.
-Nancy
[Specimen Age Sets added since February 2022_YFS_Pcod_pollock_.xlsx](https://github.com/afsc-gap-products/model-based-indices/files/11401761/Specimen.Age.Sets.added.since.February.2022_YFS_Pcod_pollock_.xlsx)

**Comment by Lewis-Barnett-NOAA** [comment]
Thanks again for these updates Nancy.  In retrospect, it would be helpful if you posted the updated spreadsheet (as you did here) each time there is a new addition.  I've updated it for our purposes through today: 
[Age_data_updates_2023.xlsx](https://github.com/afsc-gap-products/model-based-indices/files/12401598/Age_data_updates_2023.xlsx)

**Reply by NancyRoberson** [reply]
Hi Lewis,
Thanks and will do.  
Quick question:  Would it be more helpful to create new issues as ages come in or continue to add them to one single issue?

**Comment by NancyRoberson** [comment]
Hi ModSquad

Please find attached the latest age updates for Pcod, pollock and YFS.  Updates are highlighted in gold.
Give me a shout if you have any questions or suggestions.

[Age_data_updates_2023_5Sept2023.xlsx](https://github.com/afsc-gap-products/model-based-indices/files/12527911/Age_data_updates_2023_5Sept2023.xlsx)

**Comment by NancyRoberson** [comment]
Hello ModSquad,

2023 Pollock ages for the EBS shelf survey are now available in RACEBASE.  
Give me a shout if you have any questions or suggestions.
[2023 EBS Pollock_age_data_updates_19Sept2023.xlsx](https://github.com/afsc-gap-products/model-based-indices/files/12662699/2023.EBS.Pollock_age_data_updates_19Sept2023.xlsx)
